torch/onnx/symbolic_opset9.py

```diff
@@ -2942,7 +2787,15 @@ def native_layer_norm(
     two_cst = symbolic_helper._generate_wrapped_number(g, 2.0)
     eps_cst = symbolic_helper._generate_wrapped_number(g, eps)
 
-    mean = g.op("ReduceMean", input, axes_i=axes)
+    if g.opset < 18:
+        mean = g.op("ReduceMean", input, axes_i=axes)
+    else:
+        mean = g.op(
+            "ReduceMean",
+            input,
+            g.op("Constant", value_t=torch.tensor(axes, dtype=torch.long)),
+        )
+
```

Comment:
should we move this to symbolic_opset18.py or create a helper at symbolic_helper.py?

Reply:
Implementation on this method is not moved to helper because it uses opset9 add, sub, pow, sqrt. It would either introduce circular dependency or above methods need to be refactored/moved as well. For this reason, it is better to leave this method here.